tutorials/index.md

```diff
@@ -16,26 +16,14 @@ Example [Google Colab
 notebooks](https://colab.sandbox.google.com/notebooks/intro.ipynb) written in
 Python:
 
--   **Getting Started: Analyzing Census Data**
+-   [Getting Started: Analyzing Census Data](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_census_data.ipynb){:target="_blank"}
 
-    [Google Colab](https://colab.research.google.com/drive/1wYohDirOgVxvmL0d-oJRWdD6AXfAX_w1){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/analyzing_census_data.ipynb){:target="_blank"}
+-   [Case Study: COVID-19 Feature Exploration Analysis (by Google Health)](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/COVID_19_Feature_Exploration_Analysis_with_Data_Commons.ipynb){:target="_blank"}
 
--   **Case Study: COVID-19 Feature Exploration Analysis (by Google Health)**
+-   [Case Study: Analyzing Income Distribution](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_income_distribution.ipynb){:target="_blank"}
 
-    [Google Colab](https://colab.research.google.com/drive/1LLteGjXifwSsD-YsGwBnI-i96G777Q7j){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/COVID_19_Feature_Exploration_Analysis_with_Data_Commons.ipynb){:target="_blank"}
+-   [Case Study: Prevalence of Obesity in 500 US Cities](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_obesity_prevalence.ipynb){:target="_blank"}
 
--   **Case Study: Analyzing Income Distribution**
+-   [Case Study: Analyzing Genomic Data with Biomedical Data Commons](https://colab.research.google.com/github/datacommonsorg/api-python/blob/master/notebooks/analyzing_genomic_data.ipynb){:target="_blank"}
```

Comment:
Not for this PR, but this NB is missing license looks like.

Reply:
good catch!

Reply:
i fixed it directly from colab